release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,60 @@
+# Test Infra Lead Playbook
+
+## Overview of Test Infra Lead responsibilities
+
+Note: Currently, the test-infra lead has to be someone from Google GKE Engprod Team, in order to gain access to the prow cluster. This
+will change once we migrate our testing infrastructure under CNCF account.
```

Comment:
Can we get a link to https://github.com/kubernetes/test-infra/issues/5085 here?

Reply:
done